server/graphql/v2/interface/AccountWithProjects.ts

```diff
@@ -0,0 +1,57 @@
+import { GraphQLFieldConfig, GraphQLInterfaceType, GraphQLString, ThunkObjMap } from 'graphql';
+import { Sequelize } from 'sequelize';
+
+import { CollectiveType } from '../../../constants/collectives';
+import { Collective, Op } from '../../../models';
+import { GraphQLProjectCollection } from '../collection/ProjectCollection';
+import { GraphQLChronologicalOrderInput } from '../input/ChronologicalOrderInput';
+
+import { CollectionArgs, CollectionArgsType, CollectionReturnType } from './Collection';
+
+export const AccountWithProjectsFields: ThunkObjMap<GraphQLFieldConfig<Collective, Express.Request, any>> = {
+  projects: {
+    args: {
+      ...CollectionArgs,
+      term: {
+        type: GraphQLString,
+      },
+      orderBy: {
+        type: GraphQLChronologicalOrderInput,
+        defaultValue: { field: 'createdAt', direction: 'ASC' },
+      },
+    },
+    type: GraphQLProjectCollection,
+    resolve(account, args): CollectionReturnType<Collective> {
+      const where = {
+        type: {
+          [Op.ne]: CollectiveType.VENDOR,
+        },
```

Comment:
If I query `account.projects`, I expect all the items I'll get back to have the type `PROJECT` (I shouldn't get any events)